packages/lab/src/list-next/ListNext.tsx

```diff
@@ -55,7 +58,7 @@ export const ListNext = forwardRef<HTMLUListElement, ListNextProps>(
     const listId = useId(id);
     const listRef = useRef<HTMLUListElement>(null);
     const handleRef = useForkRef(listRef, ref);
-
+    const controlled = Boolean(selected || highlightedIndex);
```

Comment:
Could there be an issue if the first item was highlighted `highlightedIndex === 0` would zero being falsey accidentally make `controlled` `false`

Reply:
good point, i fixed this in the hook but forgot it here, I think I'll check for undefined here and send back controlled to the hook instead